sys/include/fs/fatfs.h

```diff
@@ -75,7 +62,7 @@ typedef struct fatfs_desc {
 } fatfs_desc_t;
 
 /**
- * info of a single opened file
+ * @brief info of a single opened file
```

Comment:
while at it, this could be changed to "FatFs file instance descriptor" for consistency with the above